src/sentry/rules/conditions/event_frequency.py

```diff
@@ -278,6 +284,39 @@ def query_hook(
         )
         return sums[event.group_id]
 
+    def get_sums(
+        self, model: int, groups: list[int], start: datetime, end: datetime, environment_id: int
+    ) -> dict[int, int]:
+        sums: Mapping[int, int] = self.tsdb.get_sums(
+            model=get_issue_tsdb_group_model(model),
+            keys=[group.id for group in groups],
+            start=start,
+            end=end,
+            environment_id=environment_id,
+            use_cache=True,
+            jitter_value=groups[0].id,
+            tenant_ids={"organization_id": groups[0].project.organization_id},
```

Comment:
Does this matter very much that I'm just grabbing the first one? Same question for the jitter value. I don't see a way to batch these without just picking one.

Reply:
In general, i think it's fine that we just choose one, if we do decide to just grab the first one, let's pull it out as a separate variable and add some safely checks around accessing the first element. I don't think the jitter value is hugely important, mostly just that it's set to something to ensure we don't run into a "thundering herd" problem.

For `tenant_ids`, i'm not sure what this is a representation of in snuba. originally when i was chatting to dan about this, he thought we'd just have to update the `keys` section. if that's the case, what you have should be okay.

---

Just curious, how are you testing all the snuba stuff locally?

Reply:
Realized that we're running this per project anyway so the org will always be the same for all the groups in here. 

Regarding testing, I just pushed a test (though it's not quite working yet).